requirements.txt

```diff
@@ -1 +1,2 @@
-scrapy
\ No newline at end of file
+scrapy
+selenium
```

Comment:
pandas need to be included here, if you're using dataframes, which you don't have to.